apps/studio/components/interfaces/Auth/ThirdPartyAuthForm/index.tsx

```diff
@@ -0,0 +1,160 @@
+import { PermissionAction } from '@supabase/shared-types/out/constants'
+import { Loader2 } from 'lucide-react'
+import { useState } from 'react'
+import toast from 'react-hot-toast'
+
+import { useParams } from 'common'
+import AlertError from 'components/ui/AlertError'
+import { FormHeader } from 'components/ui/Forms/FormHeader'
+import { useDeleteThirdPartyAuthIntegrationMutation } from 'data/third-party-auth/integration-delete-mutation'
+import {
+  ThirdPartyAuthIntegration,
+  useThirdPartyAuthIntegrationsQuery,
+} from 'data/third-party-auth/integrations-query'
+import { useCheckPermissions } from 'hooks/misc/useCheckPermissions'
+import { useFlag } from 'hooks/ui/useFlag'
+import { cn } from 'ui'
+import ConfirmationModal from 'ui-patterns/Dialogs/ConfirmationModal'
+import { AddIntegrationDropdown } from './AddIntegrationDropdown'
+import { CreateAuth0IntegrationDialog } from './CreateAuth0Dialog'
+import { CreateAwsCognitoAuthIntegrationDialog } from './CreateAwsCognitoAuthDialog'
+import { CreateFirebaseAuthIntegrationDialog } from './CreateFirebaseAuthDialog'
+import { IntegrationCard } from './IntegrationCard'
+import {
+  INTEGRATION_TYPES,
+  getIntegrationType,
+  getIntegrationTypeLabel,
+} from './ThirdPartyAuthForm.utils'
+
+export const ThirdPartyAuthForm = () => {
+  const thirdPartyAuthEnabled = useFlag('thirdPartyAuth') || false
```

Comment:
do we need the || false here?

```suggestion
  const thirdPartyAuthEnabled = useFlag('thirdPartyAuth')
```

Reply:
Fixed.